src/common/utility.cpp

```diff
@@ -575,33 +579,58 @@ bool Utility::isConflictFile(const char *name)
         bname = name;
     }
 
-    return std::strstr(bname, "_conflict-");
+    // Old pattern
+    if (std::strstr(bname, "_conflict-"))
+        return true;
+
+    // New pattern
+    if (std::strstr(bname, "(Conflict"))
```

Comment:
what about having a file called "Romeo and Juliet (Conflicts between families).pdf"?
"Git handbook (Conflict resolution chapter).txt"?

Reply:
Actually a good point